### ISSUE TITLE
Bug Report 10: Buttons and links are not labeled for accessibility

### ISSUE THREAD

**Issue by Wood3001** [Wood3001/Portfolio#19]
**Description:** 

Screen readers require a descriptive label to identify the purpose of links and buttons. Best accessibility practices suggest that these should be added to all of my navigation elements.
 
**Steps to Reproduce:**

- Visit [www.sherwoodseabrook.com](http://www.sherwoodseabrook.com/) 
- Right-click on any navigation element and select the browser’s inspector
- Inspect the <button> or <a> tag of the selected element

**Expected Result:** 

An ‘aria-label’ attribute should be present inside the tag, containing a short description of the element’s purpose.

**Actual Result:** 

There are no ‘aria-label’ attributes.
 
**Configuration:** 

This bug has been observed on all known configurations
 
**Severity:** 

High

**Priority:** 

High

**Type:** 

Bug

**Comment by Wood3001** [comment]
aria-label attributes added to all navigation elements
